### ISSUE TITLE
Implement new "any" function

### ISSUE THREAD

**Issue by mikkeschiren** [TwinProduction/gatus#71]
Hi,
I Really like Gatus, and have an question about config - as an example I have a domain that has two A records, one or the other could be the one that answers first - so how could I config gatus with that? Like adding an OR in Body response (yeah, this did not work :) )

Like: 

```
    dns:
      query-name: "mydomain.com"
      query-type: "A"
    conditions:
      - "[BODY] == 199.1.1.0||199.1.1.1"
```

**Comment by TwinProduction** [comment]
Gatus doesn't currently support the OR operator, but that's a very good feature request, as it'd have to be implemented at one point.

**Comment by mikkeschiren** [comment]
Great to hear.

**Comment by TwinProduction** [comment]
@mikkeschiren I implemented the `any` function as of 819093cb7e8d141703cf7781f10a6f3f7e2226fc

For your use case, it can be used like so:
```yaml
    dns:
      query-name: "mydomain.com"
      query-type: "A"
    conditions:
      - "[BODY] == any(199.1.1.0, 199.1.1.1)"
```

It's currently only available in `latest`.
Could you try it and let me know if it works as intended?

**Comment by mikkeschiren** [comment]
Thanks @TwinProduction  - I will try this out!

**Comment by mikkeschiren** [comment]
Worked great!

**Comment by TwinProduction** [comment]
Released in [v1.12.0](https://github.com/TwinProduction/gatus/releases/tag/v1.12.0)
